api-reference/media-api/list-and-search-files.md

```diff
@@ -750,15 +787,9 @@ imagekit = ImageKit(
     url_endpoint = 'https://ik.imagekit.io/your_imagekit_id/'
 )
 
-list_files = imagekit.list_files(options=ListAndSearchFileRequestOptions(search_query='name="file-name.jpg"'))
+list_files = imagekit.list_files({'searchQuery': 'name="file-name.jpg"'})
```

Comment:
This is invalid old code. Please take latest changes from `dev` branch. This PR diff should not show any other changes besides `.net` section.

Reply:
I have taken latest of dev branch and  still it's showing differences.